.github/workflows/ci.yml

```
@@ -56,4 +56,4 @@ jobs:
           restore-keys: ${{ github.job }}-ccache-cache
 
       - name: CI script
-        run: ./ci/test_run_all.sh
+        run: env BASE_ROOT_DIR=$GITHUB_WORKSPACE ./ci/test_run_all.sh
```

fae6642af234cad6063945bcba7958168bf2687c, style nit, suggesting to set environment variables consistently throughout the entire `ci.yml` file, wherever possible:

```suggestion
        env:
          BASE_ROOT_DIR: ${{ github.workspace }}
        run: ./ci/test_run_all.sh
```